public/js/selectors/index.js

```diff
@@ -1,35 +1,62 @@
 import moment from "moment";
-import memoize from "lodash/memoize";
-
-// If the state object is the same then just return the same obj
-export const getFilters = memoize(({ filters }) => ({
-    ...filters,
-    values: {
-        ...filters.values,
-        startDate: moment(filters.values.startDate),
-        endDate: moment(filters.values.endDate)
-    }
+import { createSelector } from "reselect";
+
+export const getIsUpdating = ({ isUpdating }) => isUpdating;
+
+/* Filters */
```

Comment:
If the functions are being separated by comments do they belong in different files instead?